Ocarina/OcarinaManager.swift

```diff
@@ -238,8 +238,13 @@ extension OcarinaManager: URLSessionDataDelegate {
         }
         
         var html: HTMLDocument? = nil
+
         if let data = data {
-            html = try? HTML(html: data, encoding: .utf8)
+            do {
+                html = try HTML(html: data, encoding: .utf8)
+            } catch {
+                self.completeRequestsWithError(error, for: originalURL)
```

Comment:
It seems like we could call `completeRequestsWithError` twice now, here and below in line 255. Is that correct?

Reply:
Yeah tbf I don't need any of these changes. I'll revert it, it was test code but I just left it. Will revert and just have the version bump only